pulseapi/users/views.py

```diff
@@ -63,7 +69,7 @@ def new_nonce_value(request):
 
 
 # API ROUTE: /nonce
-def nonce(request):
+def nonce(request, **kwargs):
```

Comment:
what are all these kwargs used for? Future devs might see this, see the function not actually accesss kwargs anywhere, and go "we can remove this"; do things not work without the kwargs argument?

Reply:
I'll add a note. We need this function to accept the version (whatever the parameter is called) even though we don't use it in the function, so I don't want to use a named param coz flake8 would complain about unused vars.